tests/test_decam.py

```diff
@@ -48,9 +48,30 @@ def test_decam_translator(self):
                            temperature=11.9*u.deg_C,
                            visit_id=229388,
                            wcs_params=dict(max_sep=1.5))),
+                     ("fitsheader-decam-0160496.yaml",
+                      dict(telescope="CTIO 4.0-m telescope",
+                           instrument="DECam",
+                           boresight_rotation_coord="unknown",
+                           dark_time=0.0407898*u.s,
+                           detector_exposure_id=None,
+                           detector_name="S1",
+                           detector_num=25,
+                           exposure_id=None,
+                           exposure_time=0.0*u.s,
+                           object="postflats-BIAS",
+                           observation_id="ct4m20121211t220632",
+                           observation_type="zero",
```

Comment:
You don't want to test `observation_type="science"`?

Reply:
The other header tested in that file is a science observation.